docs/architecture/porting-existing-aspnet-apps/migrate-aspnet-core-2-1.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Migrating to ASP.NET Core 2.1
+description: As the last .NET Core version to support .NET Framework runtime targeting, does migrating to .NET Core 2.1 make sense as an intermediate step in some app migration plans?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Migrating to ASP.NET Core 2.1
+
+ASP.NET Core 2.1 is an interesting release because it is the most recent supported .NET Core release to support both netcore and netframework runtimes. As such, it may offer an easier upgrade path for some applications when compared to upgrading all parts of the app to .NET Core at once. As an LTS release, support for .NET Core 2.1 will continue through August 2021.
```

Comment:
```suggestion
ASP.NET Core 2.1 is an interesting release because it is the most recent supported .NET Core release to support both .NET Core and .NET Framework runtimes. As such, it may offer an easier upgrade path for some applications when compared to upgrading all parts of the app to .NET Core at once. As an LTS release, support for .NET Core 2.1 will continue through August 2021.
```